ibm/service/power/resource_ibm_pi_cloud_connection.go

```diff
@@ -226,8 +236,16 @@ func resourceIBMPICloudConnectionCreate(ctx context.Context, d *schema.ResourceD
 	client := st.NewIBMPICloudConnectionClient(ctx, sess, cloudInstanceID)
 	cloudConnection, cloudConnectionJob, err := client.Create(body)
 	if err != nil {
-		log.Printf("[DEBUG] create cloud connection failed %v", err)
-		return diag.FromErr(err)
+		if vpcUnavailable.Match([]byte(err.Error())) {
+			err = retryCloudConnectionsVPC(func() (err error) {
+				cloudConnection, cloudConnectionJob, err = client.Create(body)
+				return
+			}, vpcRetryCount, vpcRetryDuration, "create", err)
```

Comment:
Any reason to pass `vpcRetryCount`, `vpcRetryDuration`? They are global const; there is no need to pass if they are the same.